spec/truefi2/BorrowingMutex.spec

```diff
@@ -49,7 +51,11 @@ rule functionDoesNotUnbanBorrower {
     env e;
     require isReasonableEnv(e);
     calldataarg args;
-    f(e, args);
+    if (f.isFallback) {
+        f@withrevert(e, args);
```

Comment:
We've decided on this pattern for fallback functions, but also for deprecated functions that will always fail and functions whose failure is determined by assumptions. This is for the sanity checks which require that assertions are reachable in all rules.